components/CreateIspoForm.js

```diff
@@ -304,8 +301,43 @@ function CreateIspoFormContent({onSubmit: _onSubmit, currentEpoch}) {
 
   const onSubmit = async (data) => {
     try {
+      const defaultTokenValues = await getDefaultTokenValues()
```

Comment:
this is a dirty workaround I came up with in hurry to compensate for the fact that the values from `SelectTokenField` form are loaded into the form state only after mounting the modal triggered after clicking on the token select element. Ideally, the form should have its fields set to their default values from the beginning and we would just access them here instead of back-filling them if they are missing, could you please have a look at it @AndrejZelonka ?